.github/workflows/ci.yaml

```diff
@@ -146,3 +152,42 @@ jobs:
             ${{ runner.os }}-build-
             ${{ runner.os }}-
       - run: maturin build -b bin
+
+  # TODO(rgerecke): What part should run on MR, and what only on main?
+  playground_build:
```

Comment:
Perhaps this should be moved to its own workflow, that runs on `main` update, and continuously releases the playground?

The _ideal_ would probably be that we build and deploy a preview URL to Pages on every PR, and that URL gets linked back to the PR via a comment or as the hyperlink on the check completion. But, that doesn't have to happen here.

Reply:
Moved it to separate workflow.

I agree on the ideal, but I would prefer to get this into a merge-able state first.

Reply:
(Sounds good, I know I'm a bit behind on this!)

Reply:
Nah, it's all good. :) I will, from time to time, rebase this to resolve conflicts, but probably not daily

Reply:
I'm gonna go ahead and resolve conflicts, then merge. There are some things I want to tweak, but better to get this merged and run from there! I'll also setup the deployment keys.